06-improve-agent-performance/improvement-overview.ipynb

```diff
@@ -19,9 +19,13 @@
         "- Evaluation Job\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print(content)
</pre><p>you can just put <code>content</code> and the notebook will display it (no print needed) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>